src/utils/vc/crypto/common/DockCryptoSignature.js

```diff
@@ -113,15 +114,12 @@ export default withExtendedStaticProperties(
       delete trimmedProof.proofValue;
 
       let credSchema;
-      if (document.credentialSchema) {
+
+      // If we already have a schema to use, add that first and then generate relaxed values later on
+      if (document.credentialSchema && document.credentialSchema.id) {
         credSchema = CredentialSchema.fromJSON({
           // Passing all the default parsing options. Ideally `document.credentialSchema` should contain these
-          parsingOptions: {
-            useDefaults: false,
-            defaultMinimumInteger: -((2 ** 32) - 1),
-            defaultMinimumDate: -((2 ** 44) - 1),
-            defaultDecimalPlaces: 0,
-          },
+          parsingOptions: DefaultSchemaParsingOpts,
```

Comment:
If `document.credentialSchema.id` is present, it means the schema is already present so the schema's parsing optons should be used and not default because they could be different.

Reply:
if they are present they will be used, if not they will use defaults...

Reply:
i think you are missing the point that at this point in code it could be a json-ld document with no parsing opts set, and they will be set after the signing

Reply:
OK